beluga_amcl/README.md

```diff
@@ -0,0 +1,66 @@
+# Beluga AMCL
+
+Beluga AMCL is a ROS 2 node based on [Beluga](../beluga) featuring interface parity with
+[nav2's AMCL][nav2_amcl].
+This package can be easily integrated with code that currently uses `nav2's AMCL`.
+
+## ROS 2 Interface
+
+### Parameters
+
+Beluga AMCL currently supports the majority of ROS parameters used in [nav2's AMCL][nav2_amcl].
+See the [nav2's configuration guide][nav2_configuration_guide] and
+[Beluga examples](../beluga_example/config/params.yaml) for reference.
+
+#### Extra Parameters
+
+Additionally, this node supports the following extra parameters:
```

Comment:
```suggestion
Additionally, the node supports the following extra parameters:
```

Reply:
Why not "this"?

Reply:
It's not wrong, but reads weird to me because it's the first paragraph in the section and we have not refered the node before under the same title. Imagine you skip right into that section through a link and start reading there.

In the same spirit, I think I would remove the "Additionally"  as well.

Reply:
Changed in https://github.com/Ekumen-OS/beluga/pull/186/commits/f81d88d7bc69c546bfde1fe4b1405d015b56c95c so it's part of the previous paragraph.